src/index.js

```diff
@@ -25,5 +26,8 @@ module.exports = function({
       eventFilter,
       stepFunction,
     }),
+    addExecutionTag: ExecutionTags.addTag,
```

Comment:
In the documentations we use `const lumigo = require('@lumigo/tracer')({ token: 'DEADBEEF' })`.
So it will allow users to do `lumigo.addExecutionTag` without splitting the code above to 2 lines:
```
const lumigoImport = require('@lumigo/tracer')
const lumigo = lumigoImport({ token: 'DEADBEEF' })
lumigoImport.addExecutionTag
```

Efi asked to add this behavior as well.